app/models/post.rb

```diff
@@ -1,5 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments
   has_many :likes
-  has_many :Comments
+
+  def increment_user_posts_count
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  after_save { increment_user_posts_count }
```

Comment:
- [optional ] Great job adding the rails active callback- `after_save` like the previous reviewer noted. Please kindly make the method you are calling private, reason been that If left public, they can be called from outside of the model and violate the principle of object encapsulation. Please implement that for all.